metals/src/main/scala/scala/meta/metals/providers/CodeActionProvider.scala

```diff
@@ -1,21 +0,0 @@
-package scala.meta.metals.providers
-
-import scala.meta.metals.WorkspaceCommand.ScalafixUnusedImports
-import scala.meta.lsp.CodeActionParams
-import scala.meta.lsp.Command
-import scala.meta.lsp.Diagnostic
-import scala.meta.lsp.Command
-import io.circe.syntax._
-
-object CodeActionProvider {
-  def codeActions(params: CodeActionParams): List[Command] = {
```

Comment:
I was tempted to keep this infrastructure but decided to remove it since it would be dead code. We can revive it once we add more code actions.